compiler/src/dmd/mars.d

```diff
@@ -1034,6 +1034,9 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
             case "gnu":
                 params.v.messageStyle = MessageStyle.gnu;
                 break;
+            case "sarif":
+                params.v.messageStyle = MessageStyle.sarif;
+                break;
             default:
                 error("unknown error style '%.*s', must be 'digitalmars' or 'gnu'", cast(int) style.length, style.ptr);
```

Comment:
also update the error message here

Reply:
done- https://github.com/dlang/dmd/pull/16986/commits/857749b98ed78340c447b682d450a2cb8fcfc2b0